access/src/main/java/edu/unc/lib/dl/ui/controller/SearchActionController.java

```diff
@@ -39,13 +38,13 @@
 /**
  * Controller which interprets the provided search state, from either the last search state in the session or from GET
  * parameters, as well as actions performed on the state, and retrieves search results using it.
- * 
+ *
  * @author bbpennel
  */
 @Controller
 public class SearchActionController extends AbstractSolrSearchController {
-    @Autowired(required = true)
-    protected PID collectionsPid;
+  //  @Autowired(required = true)
+    protected String collectionsPid = "cdr:collection";
```

Comment:
You can get the `contentRootPid` from `RepositoryPaths`, which is the current equivalent to the collections pid.  It is static, so you don't really need to store it in an instance variable

Reply:
Updated.